Docs/source/Software/Tasks.rst

```diff
@@ -2,6 +2,37 @@
 Real Time Operating System Tasks
 ********************************
 
+Initialization Task: Alex Koo
+===============================
+
+Purpose
+    Create tasks by calling "OSTaskCreate()" and provide argument specifying to RTOS how the task will be managed.
+    Initialization Task also creates the semaphores which is a programming construct that helps us achieve concurrency 
+    by implementing both synchronization and mutual exclusion. Semaphores are used when a task wants exclusive 
+    access to a resource, needs to synchronize its activities with an ISR or a task, or is waiting until an event occurs.
+    The semaphores deletes itself at the end and saves the contents of floating point registers whenever there is a context switch. 
```

Comment:
The semaphores don't delete themselves in the end, the task does. I would say that the task deletes itself after running in the timing requirements.